blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubPipelineCreateRequest.java

```diff
@@ -92,8 +93,10 @@ public BluePipeline create(Reachable parent) throws IOException {
         String singleRepo = repos.size() == 1 ? repos.get(0) : null;
 
         User authenticatedUser =  User.current();
-
-        Item item = Jenkins.getInstance().getItemByFullName(orgName);
+        
+        ModifiableTopLevelItemGroup orgRoot = getParent();
+        
+        Item item = Jenkins.getInstance().getItemByFullName(orgRoot.getFullName() + '/' + orgName);
```

Comment:
@alvarolobato  orgRoot is the root of the Blue Ocean Organisation.

@amuniz because if the pipeline is hierachicatl (ie in a sub folder) then I do not think that `getItem("folder/pipeline")` will work.  Probably needs a new API in `ModifiableTopLevelItemGroup`